docs/source/run.rst

```diff
@@ -64,6 +64,10 @@ Commonly Used Arguments
   adjust the name of Gunicorn process as they appear in the process system
   table (which affects tools like ``ps`` and ``top``).
 
+.. Gevent note::
```

Comment:
It might be problem with others APIs like eventlet.monkey_patch() but I've
never tested it until now. Have you?

On Sat, Oct 3, 2015 at 7:59 PM, Berker Peksag <notifications@github.com>
wrote:

> In docs/source/run.rst
> <https://github.com/benoitc/gunicorn/pull/1121#discussion_r41088807>:
>
> > @@ -64,6 +64,10 @@ Commonly Used Arguments
> >    adjust the name of Gunicorn process as they appear in the process system
> >    table (which affects tools like ``ps`` and ``top``).
> >
> > +.. Gevent note::
>
> Well, then the current wording needs to be improved since
> monkey.patch_all() is only true for Gevent(for example, Eventlet's API is
> eventlet.monkey_patch()). Another option is creating a new sub-section at
> http://docs.gunicorn.org/en/latest/run.html#integration and add all
> information there regarding to async workers.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benoitc/gunicorn/pull/1121/files#r41088807>.
>